tests/Integration/ApiPlatform/EndPoint/ProductEndpointTest.php

```diff
@@ -67,10 +67,11 @@ public static function tearDownAfterClass(): void
      * @param string $method
      * @param string $uri
      */
-    public function testProtectedEndpoints(string $method, string $uri): void
+    public function testProtectedEndpoints(string $method, string $uri, string $contentType): void
```

Comment:
```suggestion
    public function testProtectedEndpoints(string $method, string $uri, string $contentType = 'application/json'): void
```
Just a suggestion but you could define a default value, so only the partial update endpoint ould need to specify the different value Not blocking though